utils/crypto/bls/signers/json-rpc/server.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package jsonrpc
```

Comment:
nit: this package name is inconsistent with `localsigner`... maybe we rename `localsigner` to `local` or `jsonrpc` to `jsonrpcsigner`. Alternatively we can pull this out of the PR and put it into a subsequent one.

Reply:
🤔. If we're happy with how testing is split up, I don't mind pulling it out. The only thing is that the way I split up the testing might look a little silly with only a single implementation. 

I'm easy though, either way works. 

I also think `jsonrpcsigner` makes the most sense here and will make that change regardless.